tests/sentry/web/test_api.py

```diff
@@ -164,7 +168,11 @@ def test_organization_url_authenticated(self):
         assert data["isAuthenticated"] is True
         assert data["lastOrganization"] == self.organization.slug
         assert data["sentryUrl"] == "http://testserver"
-        assert data["organizationUrl"] == f"http://{self.organization.slug}.us.testserver"
+        assert data["links"] == {
+            "organizationUrl": f"http://{self.organization.slug}.us.testserver",
```

Comment:
Shouldn't this be `http://{self.organization.slug}.testserver`?

Reply:
I plan to address this in https://github.com/getsentry/sentry/pull/37231.